pkgs/tools/security/bitwarden_rs/default.nix

```diff
@@ -2,32 +2,31 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "bitwarden_rs";
-  version = "1.9.1";
+  version = "1.13.0";
 
   src = fetchFromGitHub {
     owner = "dani-garcia";
     repo = pname;
     rev = version;
-    sha256 = "0jfb4b2lp2v01aw615lx0qj1qh73hyrbjn9kva7zqp74wcfw12gp";
+    sha256 = "1qhfm58x08q6sv9r5nqs0qryggcyrasjlxw3h1wb3j6aq474p6jf";
   };
 
-  cargoPatches = [
-    # type annotations required: cannot resolve `std::string::String: std::convert::AsRef<_>`
-    ./cargo-lock-lettre.patch
-  ];
-
   nativeBuildInputs = [ pkgconfig ];
   buildInputs = [ openssl ] ++ stdenv.lib.optionals stdenv.isDarwin [ Security CoreServices ];
 
   RUSTC_BOOTSTRAP = 1;
 
-  cargoSha256 = "0p39gqrqdmgqhngp1qyh6jl0sp0ifj5n3bxfqafjbspb4zph3ls4";
+  cargoSha256 = "03szl83r9mjn08mqkwah8fmzpszci8ay7ki69yx1rgv3zx7npnxk";
+  cargoBuildFlags = [ "--features sqlite" ];
+
+  # FIXME: The rust check phase seems broken due to passing the flags after --.
+  doCheck = false;
```

Comment:
Well, the right fix would be to fix `checkPhase` instead, because it does not really make sense to add flags after `--`. Apparently some flags can be added after `--` (seems a weird API design choice to me) as can be seen in the `cargo test` manpage (e.g. `cargo test -- --nocapture`), but the documented flags have to be added before `--`, like `--features`. Maybe it can be fixed by adding `checkArgs` to differentiate between flags and arguments.

Reply:
Just passing all cargo flags to cargo test could solve this issue.